Readme.md

```diff
@@ -77,6 +79,66 @@ model = ORTModule(model)
 
 To see torch-ort in action, see https://github.com/microsoft/onnxruntime-training-examples, which shows you how to train the most popular HuggingFace models.
 
+# Accelerate inference for PyTorch models with ONNX Runtime (Preview)
+
+ONNX Runtime for PyTorch accelerates PyTorch model inference using ONNX Runtime.
+
+It is available via the torch-ort-inference python package. This preview package enables OpenVINO™ Execution Provider for ONNX Runtime by default for accelerating inference on various Intel CPUs and integrated GPUs.
+
+This repository contains the source code for the package, as well as instructions for running the package.
+
+## Install in a local Python environment
+
+By default, torch-ort-inference depends on PyTorch 1.12 and ONNX Runtime OpenVINO EP 1.11.
+
+Install torch-ort-inference
+
+- `pip install torch-ort-inference`
+
+## Verify your installation
+
+Once you have created your environment, using Python, execute the following steps to validate that your installation is correct.
+
+1. Download a inference script
+
+   - `wget https://raw.githubusercontent.com/pytorch/ort/main/torch_ort_inference/tests/bert_for_sequence_classification.py`
+
```

Comment:
Currently I don't see the "tests" directory yet. Will this be added later along with the samples?

Reply:
That's right.